code/api/src/modules/user/model.js

```diff
@@ -14,6 +14,9 @@ module.exports = function(sequelize, DataTypes) {
     },
     role: {
       type: DataTypes.TEXT
+    },
+    description: {
+      type: DataTypes.TEXT
     }
```

Comment:
This block of code prevents the user from being able to log in. The pull request that documents this change does not use the template and lacks detailed documentation of the changes made so the front end team is unable to know whether or not the backend team was aware of this change in functionality.

Reply:
> This block of code prevents the user from being able to log in. The pull request that documents this change does not use the template and lacks detailed documentation of the changes made so the front end team is unable to know whether or not the backend team was aware of this change in functionality.

I agree the template should've been used. I merged this and as such I take full responsibility for not seeing that and correcting the issue at that time. My apologies to all.

As for this block preventing the user from logging in, would you mind expanding on that? Was an error thrown? Was logging in simply not happening? Thanks in advance